packages/forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -0,0 +1,91 @@
+const fetch = require('node-fetch');
```

Comment:
This file and related should be removed?

Reply:
done